packages/http-server-csharp/src/service.ts

```diff
@@ -557,6 +569,7 @@ export async function $onEmit(context: EmitContext<CSharpServiceEmitterOptions>)
         ifaceName,
         code`${this.#licenseHeader}
       // <auto-generated />
+      #nullable enable
```

Comment:
does this mean we add nullable enable in all generated files?

Reply:
Generally in any file where we could use nullables, yes.  Ideally we would only use this if we actually had references to nullable reference types in a particular file.